qt/python/resources.qrc

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE RCC><RCC version="1.0">
+<qresource>
+    <file alias="MantidPlot_Icon_32offset.png">../../images/MantidPlot_Icon_32offset.png</file>
+    <file alias="data_replace.png">../../images/data_replace.png</file>
+    <file alias="fileclose.png">../../images/fileclose.png</file>
+    <file alias="history.png">../../images/history.png</file>
```

Comment:
I'm still getting a `QPixmap::scaleHeight: Pixmap is a null pixmap` warning. I think we are missing the `history_off.png` image that is used by https://github.com/mantidproject/mantid/blob/master/qt/widgets/common/src/PropertyWidget.cpp#L372.